elastic/kibana
# Make Task Manager internals use bulkRemove to delete tasks

In this PR, I'm making the buffered task store `remove` function actually buffer and delete tasks in bulk requests. I am also optimizing the task store code to not wait for an Elasticsearch refresh when doing a remove call. This will make Task Manager run up to 1s faster per task when the task is ad-hoc (ex: actions).

## To verify

1. Create a rule that fires actions (ex: server log).
2. Once the actions execute, 1) ensure no errors are logged 2) verify that the task document has been removed from the task manager index.
